### PR TITLE
Update README for CLAWdit

### PR DESCRIPTION
**Pull request by manez**
**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/1267

# What does this Pull Request do?

Updates the readme to match the [template](https://github.com/Islandora-CLAW/CLAW/blob/master/docs/contributing/readme_template.md)

# What's new?

A few headers renamed, a few sections added.

# How should this be tested?

Preview the page and check it against the [template](https://github.com/Islandora-CLAW/CLAW/blob/master/docs/contributing/readme_template.md)

# Interested parties
@rangel35 want to review one of mine? 😸

### CODE REVIEW

**Comment by manez** [comment]
Or maybe @whikloj since you're the maintainer now?

File: README.md [hunk 1]
```diff
@@ -63,16 +63,37 @@ Modify the [example configuration](./conf/syn-settings.example.xml) and move it
 ### Header principals
 Additional roles are passed to Fedora via a HTTP header, this is configured via the `header` attribute to the `<config>` element in the syn-settings.xml.example file. You must also configure Fedora to read this header via its HeaderProvider.
 
+## Documentation
+
+Further documentation for this module is available on the [Islandora 8 documentation site](https://islandora-claw.github.io/CLAW/).
```
Comment (whikloj) [inline README.md]: This new to change to the new documentation site.

Comment (whikloj) [reply README.md]: https://islandora.github.io/documentation/

File: README.md [hunk 2]
```diff
@@ -63,16 +63,37 @@ Modify the [example configuration](./conf/syn-settings.example.xml) and move it
...
 ## Development
 
 If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
 
 If you would like to contribute code to the project, you need to be covered by an Islandora Foundation [Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_cla.pdf) or [Corporate Contributor Licencse Agreement](http://islandora.ca/sites/default/files/islandora_ccla.pdf). Please see the [Contributors](http://islandora.ca/resources/contributors) pages on Islandora.ca for more information.
 
-## Licensing
+## Development
+
+If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
```
Comment (whikloj) [inline README.md]: Tech call url should be https://github.com/islandora/documentation/wiki

**Comment by manez** [comment]
@whikloj updated! nothing left but the playbook now

**Pull request merged**